src/Mvc/test/Mvc.FunctionalTests/SimpleWithWebApplicationBuilderTests.cs

```diff
@@ -1,31 +1,96 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using System.Net;
-using System.Net.Http;
+using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Mvc.FunctionalTests
 {
     public class SimpleWithWebApplicationBuilderTests : IClassFixture<MvcTestFixture<SimpleWebSiteWithWebApplicationBuilder.FakeStartup>>
     {
+        private readonly MvcTestFixture<SimpleWebSiteWithWebApplicationBuilder.FakeStartup> _fixture;
+
         public SimpleWithWebApplicationBuilderTests(MvcTestFixture<SimpleWebSiteWithWebApplicationBuilder.FakeStartup> fixture)
         {
-            Client = fixture.CreateDefaultClient();
+            _fixture = fixture;
         }
 
-        public HttpClient Client { get; }
-
         [Fact]
         public async Task HelloWorld()
         {
             // Arrange
             var expected = "Hello World";
+            using var client = _fixture.CreateDefaultClient();
+
+            // Act
+            var content = await client.GetStringAsync("http://localhost/");
+
+            // Assert
+            Assert.Equal(expected, content);
+        }
+
+        [Fact]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/33889")]
```

Comment:
I wasn't sure if this was the right syntax or not for a test known to be failing due to an active issue so it gets ignored and doesn't break the build.